src/dbnode/storage/shard.go

```diff
@@ -2663,21 +2659,24 @@ func (s *dbShard) Repair(
 
 func (s *dbShard) AggregateTiles(
 	sourceNsID ident.ID,
-	sourceShardID uint32,
+	targetNs Namespace,
+	shardID uint32,
 	blockReaders []fs.DataFileSetReader,
 	writer fs.StreamingWriter,
 	sourceBlockVolumes []shardBlockVolume,
 	opts AggregateTilesOptions,
-	targetSchemaDescr namespace.SchemaDescr,
 ) (int64, error) {
 	if len(blockReaders) != len(sourceBlockVolumes) {
-		return 0, fmt.Errorf("blockReaders and sourceBlockVolumes length mismatch (%d != %d)", len(blockReaders), len(sourceBlockVolumes))
+		return 0, fmt.Errorf(
+			"blockReaders and sourceBlockVolumes length mismatch (%d != %d)",
+			len(blockReaders),
+			len(sourceBlockVolumes))
 	}
 
 	openBlockReaders := make([]fs.DataFileSetReader, 0, len(blockReaders))
 	defer func() {
 		for _, reader := range openBlockReaders {
-			reader.Close()
+			_ = reader.Close()
```

Comment:
nit; may be worth logging these? If they'd be too noisy on errors here, add a comment to that effect

Reply:
Will add logging, should not be noisy at all.